homeassistant/components/tuya/fan.py

```diff
@@ -118,16 +116,13 @@ def is_on(self):
         return self._tuya.state()
 
     @property
-    def speed(self) -> str:
+    def percentage(self) -> str:
```

Comment:
```py
def percentage(self) -> Optional[int]:
```

Reply:
Oops. Thanks!

Reply:
fixed in https://github.com/home-assistant/core/pull/46541